Python/Product/VSInterpreters/PackageManager/PipPackageManager.cs

```diff
@@ -110,6 +112,19 @@ int priority
             }
         }
 
+        private void EnsureActivated() {
```

Comment:
For miniconda 4.5.11 and before, the base miniconda was not required to be activated before some operations could be performed on it. Now that we're activating the base and then performing those operations, could this cause backward compatibility issues with 4.5.11-?

Reply:
We were supposed to activate with 4.5.11, but it wasn't failing if we didn't. I double-checked again and it is working properly.